UNRELEASED.md

```diff
@@ -1,3 +1,5 @@
 # Unreleased
 
-_Nothing yet..._
\ No newline at end of file
+**Added:**
```

Comment:
This needs to be "Breaking". Also can you mention the `fetchFunction` prop was replaced in favour of the new data source approach?

Reply:
Changed